### PR TITLE
fix(environment): Create POSTGRES_DATABASE_NAME env variable and fix GITLAB_URL

### LINKED ISSUE #28: GITLAB_URL not loaded properly
When using the docker image internally at my company, the [GITLAB_URL](https://github.com/ManoManoTech/homer/blob/main/src/constants.ts#L10) does not seem to load properly.

Steps to reproduce on internal Gitlab:
- Pull image from GitHub and build another one in internal Gitlab.
- Run the image on Gitlab pipeline with all env variables set.

An error in log mentioning `Environment variable GITLAB_URL not found`

### LINKED ISSUE #29: DB name is hardcoded
The [database name](https://github.com/ManoManoTech/homer/blob/main/src/core/services/data.ts#L23) is hardcoded.

How to fix:
- Use an environment variable instead.

### PR DESCRIPTION
**Pull request by cicoub13**
Fix #29
Fix #28

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
+POSTGRES_HOST=localhost
 POSTGRES_PASSWORD=root
 POSTGRES_PORT=54320
 POSTGRES_USER=root
+POSTGRES_DATABASE_NAME=homer
+SLACK_BOT_USER_O_AUTH_ACCESS_TOKEN=SLACK_BOT_USER_O_AUTH_ACCESS_TOKEN
+SLACK_SIGNING_SECRET=SLACK_SIGNING_SECRET
+EMAIL_DOMAINS=my-domain.com,ext.my-domain.com
+GITLAB_URL=https://my-git.domain.com
+TICKET_MANAGEMENT_URL_PATTERN=https://my-ticket-management.com/view/{ticketId}
```
Comment (greg0ire) [inline .env.development:13]: Please configure your IDE to always add a newline at end of file, this will make the red pictogram disappear.

Comment (greg0ire) [inline .env.development:13]: Should we use the alphabetical order here? It helps avoiding duplicates or conflicts.

File: README.md [hunk 2]
```diff
@@ -227,6 +227,7 @@ the following variables:
 - `POSTGRES_PASSWORD`: password used when connecting to the database.
 - `POSTGRES_PORT`: port where the database listens.
 - `POSTGRES_USER`: username used when connecting to the database.
+- `POSTGRES_DATABASE_NAME`: database name (like `homer`).
```
Comment (greg0ire) [inline README.md:230]: Let's preserve the alphabetical order here?

Comment (cicoub13) [reply README.md]: Fixed

File: .env.development (hunk 1 above)
Comment (cicoub13) [reply .env.development]: Good idea, done

Comment (cicoub13) [reply .env.development]: Fixed

**Pull request merged**